build-livecd-root

```diff
@@ -1,95 +1,10 @@
 #!/bin/bash -x
 
-# the portion of building a livecd that needs to be run as root
-# run this after running build-livecd
-#
-# The temp dir in which things happen is by default deleted when this
-# script ends, unless an error happens or the KEEP_TMPDIR variable is set
-
-function cleanup() {
-    if [ -n "$KEEP_TMPDIR" ]; then
-        echo "Not removing tmpdir $tmpdir"
-    else
-        [ -d $tmpdir ] && rm -rf $tmpdir
-    fi
-}
-
-if [ $(id -u) != 0 ]; then
-  echo "Please run this script as root"
-  exit 1
+rm /var/tmp/lmc-work-* -rf
+if [ ! -e CentOS-8.4.2105-x86_64-dvd1.iso ]; then
+    wget http://ftp.osuosl.org/pub/centos/8.4.2105/isos/x86_64/CentOS-8.4.2105-x86_64-dvd1.iso
 fi
-
-which livecd-creator isohybrid implantisomd5 >/dev/null || ( echo "Command(s) missing, install required tools" && exit 2 )
```

Comment:
Can you keep this dirty test for missing tools? Obviously check only the relevant ones.